src/main/java/com/example/App.java

```diff
@@ -8,6 +8,6 @@ public class App
 {
     public static void main( String[] args )
     {
-        System.out.println( "Hello World! - TESTing13" );
+        System.out.println( "Hello World! - TESTing14" );
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>')' is preceded with whitespace.